index.html

```diff
@@ -1336,8 +1331,10 @@ <h4>
       <a>NDEFMessage</a> interface:
     </p>
     <pre class="idl">
-      [Constructor(NDEFMessageInit messageInit), Exposed=Window]
+      [Exposed=Window]
       interface NDEFMessage {
+        constructor(NDEFMessageInit messageInit);
```

Comment:
Just a question, is this a new canonical way to declare constructors?

Reply:
This is the new way according to respect error. Just open the current spec and check the error which goes to a section explaining it

Reply:
Right, I noticed this in other specs (WoT spec broken, too), that's why the question.